### PR TITLE
feat(billing): autorenew service in debt warning

### PR DESCRIPTION
**Pull request by JeremyDec**
## Autorenew debt warning
Add warning for service in debt, before deletion.
Popup to redirect to billing history page

MFRWW-742

### CODE REVIEW

File: client/app/account/billing/autoRenew/billing-autoRenew.controller.js [hunk 1]
```diff
@@ -384,7 +384,12 @@ angular.module("Billing.controllers").controller("Billing.controllers.AutoRenew"
         $scope.resiliateService = function (service) {
             const serviceToResiliate = getServiceToResiliate(service);
 
-            $scope.setAction("delete", _.clone(serviceToResiliate, true), "autoRenew");
+            // If service has debt
```
Comment (euhmeuh) [inline client/app/account/billing/autoRenew/billing-autoRenew.controller.js]: Redundant comment :)

Comment (JeremyDec) [reply client/app/account/billing/autoRenew/billing-autoRenew.controller.js]: For sure :)

**Comment by ovh-cds** [comment]
CDS Report build-yarn#1243.0 ✘  
* Stage 1
  * deps ✘

File: client/app/common/translations/Messages_cs_CZ.xml [hunk 2]
```diff
@@ -136,6 +136,7 @@
    <translation id="common_ipv6" qtlid="34126">IPv6</translation>
    <translation id="common_bitFormat" qtlid="122107">bitů</translation>
    <translation id="common_expired" qtlid="257941">Služba nebyla obnovena. To je způsobeno Vaším rozhodnutím, nebo opožděním platby na Vaší straně.</translation>
+   <translation id="common_settle_bill">Settle</translation>
```
Comment (antleblanc) [inline client/app/common/translations/Messages_cs_CZ.xml]: Must be done on our internal tool. [Only `fr_FR.xml`](https://github.com/ovh-ux/ovh-manager-dedicated/blob/master/.github/CONTRIBUTING.md#translations) files can be updated.

Comment (antleblanc) [reply client/app/common/translations/Messages_cs_CZ.xml]: I can't find `common_settle_bill` in `Messages_**_**.xml`.

**Pull request merged**